app/views/pages/_publishing_from_rss_guide_text.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="container article" id="editor-help-guide">
+  <div class="title help-guide-title">
+    <h1>Publishing from RSS Guide 📰</h1>
+  </div>
+  <div class="body">
+    <h2>Front Matter</h1>
+    <p>Our RSS reader converts some tags from the RSS items to tags for the Jekyll front matter:</p>
+    <ul>
+      <li><code>&lt;title&gt;</code>: <strong>title</strong></li>
+      <li><code>&lt;category&gt;</code>: <strong>tags</strong><br />Seperate tags by a colon.<br />Since only 4 tags can be used in a post on dev.to, the first four tags will be used from the RSS item.</li>
+      <li><code>&lt;link&gt;</code>: <strong>canonical_url</strong><br />Only if the "Mark the RSS source as canonical URL by default" option is enabled in the Publishing from RSS settings.</li>
+    </ul>
+
+    <h2>Article Content</h2>
+    <p>The HTML in the <code>&lt;description&gt;</code>, <code>&lt;summary&gt;</code>, or <code>&lt;description&gt;</code> tag will be converted to Markdown.</p>
+    <p>Tweet links, YouTube, and Gist iframes are converted to Liquid tags.</p>
```

Comment:
@maestromac is it intentional that liquid tags only work for medium posts?

Reply:
Yes. It was a quick implementation and at the time most feeds were Medium's feed. And because it's Medium's feed, those iframe structures are consistent and can be accounted for.